.github/workflows/deploy.yml

```diff
@@ -3,9 +3,19 @@ name: Deploy to Gov PaaS
 on:
   workflow_dispatch:
     inputs:
-      deploy_to_dev:
+      environment:
+        description:  Which AWS Account to use
+        type: choice
+        required: true
+        options:
+        - test
+        - uat
+      copilot:
+        description: Whether to deploy
```

Comment:
Can we be more specific in this case? Purely on the grounds of making sure this isn't activated by accident